### DISCUSSION TITLE
VSCode Copilot Chat window is blank

### DISCUSSION THREAD

**Discussion by robutler** [community/community#62992] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I've been using Github Copilot chat for a couple of months without issue, however I haven't used my computer for a few weeks, and when I came back to it the Copilot Chat window in VSCode is just blank.

![Screenshot 2023-08-07 at 11 39 16](https://github.com/community/community/assets/7031368/438436ee-9419-4e54-ad26-b1cc8e6d3753)

I tried removing both the Copilot and the Copilot chat extension and installing them again, signing out from the GitHub account and signing in again, resetting the windows etc, to no avail.

Looking at the output from the chat extension it looks like this:
```
[INFO] [auth] [2023-08-07T09:31:10.810Z] Logged in as ***
[INFO] [default] [2023-08-07T09:31:11.483Z] copilot token chat_enabled: true
[INFO] [auth] [2023-08-07T09:31:11.484Z] Got Copilot token for ***
```

Is there anything else I can try to get this working again?

My system:
Visual Studio Code Insiders v1.82.0
GitHub Copilot v1.101.317
GitHub Copilot Chat v0.7.2023080701
macOS 13.4

**Comment by rebepoi** [comment]
Same here. For me it was working last week normally. Work aroud for now, at least for me, is to open it thrue command menu and write there chat: open editor (GitHub Copilot)

**Reply by robutler** [reply]
Nice workaround! That method worked for me as well. It's not ideal, but at least it will allow me to access the chat again.

**Comment by robutler** [comment, marked as answer]
I upgrade to the latest Insiders version, and now the chat window works again.
As can also be seen in this thread: https://github.com/microsoft/vscode-copilot-release/issues/343
